ai-service/src/main/java/org/libertybikes/ai/service/AIRoundService.java

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ */
+package org.libertybikes.ai.service;
+
+import java.net.URL;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.libertybikes.ai.restclient.LibertyBikesClient;
+
+@ApplicationScoped
+@Path("open")
+public class AIRoundService {
+
+    // TODO - url prop not working
+    @Inject
+    @RestClient
+    LibertyBikesClient client;
+
+    private LibertyBikesClient getClient() {
+        try {
+            return RestClientBuilder.newBuilder().baseUrl(new URL("http://localhost:8080")).build(LibertyBikesClient.class);
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @GET
+    @Produces(MediaType.TEXT_HTML)
+    public String doPost() {
```

Comment:
looks like this was just for debugging -- is this still needed? Also, the method is called `doPost` but it's a GET method